build.sbt

```diff
@@ -114,6 +114,8 @@ lazy val testDependencies = Seq(
   "org.scalatest" %% "scalatest" % "3.0.7",
   "org.scalactic" %% "scalactic" % "3.0.7",
   "org.scalamock" %% "scalamock" % "4.1.0",
+  "org.mockito" % "mockito-core" % "2.18.0",
```

Comment:
can you bump it to newer version?
`"org.mockito" % "mockito-scala_2.12" % "1.3.1",`